test/e2e/async-modules-app/next.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {}
```

Comment:
Could we drop this file?

Reply:
We could but most tests have one so it shouldn't matter much 👍